src/WebJobs.Script/Utility.cs

```diff
@@ -654,6 +654,11 @@ internal static int GetStableHash(string value)
             }
         }
 
+        public static string BuildStorageConnectionString(string accountName, string accessKey)
+        {
+            return $"DefaultEndpointsProtocol=https;AccountName={accountName};AccountKey={accessKey}";
```

Comment:
Not sure if this will require antares changes, but storage connection strings can optionally contain an `EndpointSuffix=` parameters. it defaults to `core.windows.net`, but for national clouds it's different.

Reply:
Yeah. looks like this is the only suffix supported for now. the control plane api doesn't provide a way to specify this. I remember seeing a todo related to this and will follow up on any plans to support other types of accounts. For now we just assume the suffix doesn't change.

Reply:
sounds god